test/test_build.py

```diff
@@ -257,8 +262,8 @@ def callp(cmd):
             callp('echo Ruby version: $(ruby -v)')
         ])
 
-    def test_it_strips_and_quotes_ruby_version(self, mock_get_logger, mock_run, patch_clone_dir):
-        version = '  $2.3  '
+    def test_it_strips_ruby_version(self, mock_get_logger, mock_run, patch_clone_dir, mock_ruby_ver):  # noqa: E501
+        version = '  2.7  '
```

Comment:
I'm guessing this test is in here for a reason, and it appears we're no longer stripping it....